src/main/java/org/ow2/proactive/model/CatalogObject.java

```diff
@@ -32,34 +32,34 @@
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.RequiredArgsConstructor;
-import lombok.Setter;
 
 
+/**
+ * This is catalog object model
+ */
 @Getter
 @RequiredArgsConstructor
 @AllArgsConstructor
 public class CatalogObject {
 
-    private final int bucketID;
+    private final int bucket_id;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "bucket_id" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS00116)